<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>912</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>930</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>1091</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Обучение классификатора</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="train_btn">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Обучить классификатора</string>
     </property>
    </widget>
    <widget class="QComboBox" name="file_combo">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>75</y>
       <width>170</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>data_500.csv</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>data_1000.csv</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>data_2500.csv</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>data_5000.csv</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>361</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Файл лля обучения модели:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="update_btn">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>140</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Обновить выбранный файл</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>701</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Метод классификации новостных текстов по тематикам
с использованием опорных векторов</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>1081</width>
      <height>571</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="title">
     <string>Классификация текстов</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>501</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Введите новостный текст или загрузите из файла (*.txt):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Тематик исходного текста:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="use_btn">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>390</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Определить тематик</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="input_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>1051</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="res_text">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>470</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="file_btn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выберите файл</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
